conda/recipes/cudf_kafka/meta.yaml

```
@@ -19,12 +19,25 @@ build:
   script_env:
     - PARALLEL_LEVEL
     - VERSION_SUFFIX
+    - CMAKE_GENERATOR
+    - CMAKE_C_COMPILER_LAUNCHER
+    - CMAKE_CXX_COMPILER_LAUNCHER
+    - CMAKE_CUDA_COMPILER_LAUNCHER
+    # TODO: remove "gha" suffix before PR is merged
+    - SCCACHE_S3_KEY_PREFIX=libcudf-aarch64-gha # [aarch64]
+    - SCCACHE_S3_KEY_PREFIX=libcudf-linux64-gha # [linux64]
```

Leave this suggestion open until we're ready to merge, then apply it.
```suggestion
    - SCCACHE_S3_KEY_PREFIX=libcudf-aarch64 # [aarch64]
    - SCCACHE_S3_KEY_PREFIX=libcudf-linux64 # [linux64]
```

---

resolved in 85b1ee1